### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by jesperin**
A post about the life of a software designer at Solita.

### CODE REVIEW

File: _posts/2017-03-15-being-a-software-designer-at-solita.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## No bargaining on tools and ergonomics
+
+To be able to efficiently do the thing you love to do one needs the right tools. The employee may choose the computer and operating system of liking plus accessories. The computer and phone are in use for 2 years and then replaced by new ones. If you need a new program then it’s easy for you to make an order and the license key arrives shortly. The employees at Solita are trusted and there is a striving not to waste valuable time on paperwork or slow processes.
```
Comment (Juusoh) [inline _posts/2017-03-15-being-a-software-designer-at-solita.md]: program --> software?

File: _posts/2017-03-15-being-a-software-designer-at-solita.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Humane resourcing - not just biomass
+
+How does a project find it’s people for implementation? Before people are chosen to a specific project the designer and the project responsible discuss about a project opening possibility. If the discussion is positive and both parties think the project is suitable then the designer has time to think about the case if all other suitability criteria are met.
+
+For people who have finished a project and are available, ready and set for next tasks, projects with available seats introduce themselves so that people get a picture of what it is they are implementing. The goal is to find the right person for the right project and make sure that the designer is feeling comfortable with the match and upcoming environment. For me the projects that I’ve been participating in have been really good matches and the role has been just what I’ve wanted at that time. So I have true possibility to make a difference in which direction my professional career is going and what kind of project and role would be most interesting and rewarding. Too good to be true? I’m certain that both the employee and employer benefits from this arrangement.
```
Comment (Rinorragi) [inline _posts/2017-03-15-being-a-software-designer-at-solita.md]: for next tasks --> for the next tasks?

File: _posts/2017-03-15-being-a-software-designer-at-solita.md (hunk 1 above)
Comment (Rinorragi) [inline _posts/2017-03-15-being-a-software-designer-at-solita.md]: 2 years --> two years?

File: _posts/2017-03-15-being-a-software-designer-at-solita.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+* knowledge communities: a community open for everyone and members share a common interest on a certain skill
+* self study: independently become acquainted with a certain area of interest or skill
+* flash of information: a short information sharing session that is open to all which is not necessarily tied to a knowledge community
+* dev meetup: all Java developers gather once in a month and team members share experiences, best practices from projects and get a shoulder to cry on if things have been rough
+
+Employees are invited to a personal development discussion by the immediate superior at least three times a year. If you like to talk more often then it’s also possible. Most of the time your superior is available, easy to approach and always likes to talk to you. The development discussion is a good opportunity to tell how you feel about the project you are working on, what skills you would like to improve and what you would like to do next. I especially like the walking type of discussion. Walk around Töölö bay on a sunny summer day, have a nice talk with your superior and maybe stop and have an ice-cream on the way. How does that sound to you?
```
Comment (Rinorragi) [inline _posts/2017-03-15-being-a-software-designer-at-solita.md]: three times a year --> two times a year? Not sure though.

**Comment by Rinorragi** [comment]
As an overview blog post was nice. I would like to see few pictures added :)

File: _posts/2017-03-15-being-a-software-designer-at-solita.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+- Solita
+- professional development
+- culture
+- development
+---
+I have been working for three years now as a Software Designer at Solita. During this time I have been involved in six different projects and I’m happy to say that they’ve all been different in regards of nature and my own role. I personally like change once in a while, but in the end it’s up to you what you like to do. In this post I will tell you how I experience Solita from the Software Designer’s perspective.
+
+## Humane resourcing - not just biomass
+
+How does a project find it’s people for implementation? Before people are chosen to a specific project the designer and the project responsible discuss about a project opening possibility. If the discussion is positive and both parties think the project is suitable then the designer has time to think about the case if all other suitability criteria are met.
```
Comment (Kauko) [inline _posts/2017-03-15-being-a-software-designer-at-solita.md]: its*. It's = it is

File: _posts/2017-03-15-being-a-software-designer-at-solita.md (hunk 1 above)
Comment (Kauko) [inline _posts/2017-03-15-being-a-software-designer-at-solita.md]: Mielestäni "of their liking".

**Comment by jesperin** [comment]
Added images and made some text changes according to comments.

**Comment by Rinorragi** [comment]
It looks nice for me. If you change date of the file I can then merge if you wish.

**Pull request merged**